s3-source-connector/build.gradle.kts

```diff
@@ -22,6 +22,8 @@ val amazonS3Version by extra("1.12.729")
 val amazonSTSVersion by extra("1.12.729")
 val s3mockVersion by extra("0.2.6")
 val parquetVersion by extra("1.14.3")
+val kafkaVersion by extra("3.6.0")
+val confluentKafkaVersion by extra("7.6.0")
```

Comment:
I bumped kafka version to 3.7.1 instead of 3.8.1 as there is no corresponding confluent version. Initially we set the version to 3.6.0 as Aiven kafka version is around that one. But as this is only for tests, we can bump to even higher.
Thanks for the suggestion.